src/test/bash/checkResources.sh

```diff
@@ -7,14 +7,9 @@ CONTENTS=`jar tvf ${JAR}`
 
 EXPECTED="Linux/aarch64/libblst.so
 Linux/amd64/libblst.so
```

Comment:
Think should add here `Linux/aarch64/libblst.so` . Wonder why there is no CI failure.